tools/flang1/flang1exe/func.c

```diff
@@ -2819,9 +2819,13 @@ rewrite_func_ast(int func_ast, int func_args, int lhs)
     }
     is_icall = FALSE;
     goto ret_call;
+  case I_GETARG:
   case I_GET_COMMAND:
   case I_GET_COMMAND_ARGUMENT:
-    if (optype == I_GET_COMMAND) {
+    if (optype == I_GETARG) {
+      rtlRtn = RTE_getarga;
```

Comment:
Does this change mean that `getarg` won't be called anymore?

Reply:
Yes, we hope to call `getarga` instead of `getarg`

Reply:
> Does this change mean that `getarg` won't be called anymore?

Yes, we hope to call `getarga` instead of `getarg`